### PR TITLE
Test for profiling RBD performance

### PR DESCRIPTION
**Pull request by keesturam**
I'm sending this WIP PR to seek feedback on the perf charts being generated.  
Link to perf report: https://docs.google.com/spreadsheets/d/1jqypI0I4Ym1lvjSKMEcWPw8mFgvgQ2cTcGFMq4u0TKs/edit#gid=0

Once the format is finalized, we can do similar profiling against all IO profiles and volume types. 

To do:
- Validate any performance deviation from baseline

### CODE REVIEW

File: ocs_ci/ocs/resources/pod.py [hunk 1]
```diff
@@ -198,20 +184,15 @@ def exec_ceph_cmd(self, ceph_cmd, format='json-pretty'):
             return [item for item in out if item]
         return out
 
-    def get_storage_path(self, storage_type='fs'):
+    def get_mount_path(self):
```
Comment (ebenahar) [inline ocs_ci/ocs/resources/pod.py]: why have you removed the option to return the block device path? This is being used in some tests

Comment (keesturam) [reply ocs_ci/ocs/resources/pod.py]: I had tried this out and seem to have sent it without reverting to the original function. I'll fix this.

File: tests/e2e/test_performance_fio_randread_profile.py [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+                *['10k', 'ro', 2, 300, 4, 1]
+            )
+        ]
+    )
+    @pytest.mark.usefixtures(base_setup.__name__)
+    def test_perf_read_latency(
+        self, size, io_direction, jobs, runtime, depth, sheet_index
+    ):
+        """
+        Test IO
```
Comment (SvetlanaRedHat) [inline tests/e2e/test_performance_fio_randread_profile.py]: Function does not have enough description,i see it does a lot  of actions and it  will be better to have description and pass and fail criteria's

File: ocs_ci/utility/spreadsheet/spreadsheet_api.py [hunk 3]
```diff
@@ -28,18 +28,45 @@ def __init__(self, sheet_name, sheet_index):
...
+        """
+        Updates the sheet name
+        """
+        return self.sheet.update_title(title)
+
+    def append_row(self, value):
+        """
+        Appends a row with content
+        """
+        return self.sheet.append_row(value)
```
Comment (RazTamir) [inline ocs_ci/utility/spreadsheet/spreadsheet_api.py]: I think that insert row is doing append of a row

File: tests/e2e/test_performance_fio_randread_profile.py (hunk 2 above)
Comment (keesturam) [reply tests/e2e/test_performance_fio_randread_profile.py]: I've added a description now. These tests as of today just generate graphs for the IO profiles. I'll introduce a check to fail the test once we have a benchmark to compare. I've added a todo for now.

**Comment by ebenahar** [comment]
@keesturam Would like to close old PRs, is this one still relevant?

**Comment by keesturam** [comment]
Closing this PR. We now use Ripsaw with FIO to run the workload.

**Pull request closed without merging**